src/components/ui/Table.tsx

```diff
@@ -13,26 +13,46 @@ import { cn } from "@/lib/utils/cn"
 const tableVariants = tv({
```

Comment:
I see what you are doing here, but this is losing the point of this utility. We are repeating the same styles many times.

The changes that you've made explain that the product table is very different from the other tables that we have.

Are we sure that we can't override or share some of the base styles that we had?

I don't feel that we need to block this PR for that reason, but I do feel that we should do something a little simpler. I'll take a closer look at it later. Maybe this is also a good opportunity to discuss with the designers and align these styles with the rest.